models/Program.js

```diff
@@ -168,7 +169,7 @@ Program.schema.virtual('isLiveNow').get(function () {
 	// Check biweekly first, then do a general bounds check.
 	if (this.isBiweekly) {
 		const curWeek = Math.floor(Math.floor(daysSinceBeginningOfYear() / 7) / 2);
-		const state = (curWeek == 0) ? true : false;
```

Comment:
I blame myself (git blame does too). At least I did this late at night so I can shift the blame a little bit :stuck_out_tongue: